playbooks/common/openshift-master/config.yml

```diff
@@ -21,6 +21,7 @@
           hostname: "{{ openshift_hostname | default(None) }}"
           public_hostname: "{{ openshift_public_hostname | default(None) }}"
           deployment_type: "{{ openshift_deployment_type }}"
+          use_flannel: "{{ openshift_use_flannel | default(None) }}"
```

Comment:
This shouldn't be needed here, because the flannel_register role is assigned to the host after the openshift_master role is, so openshift_common will have set the openshift.common.use_flannel variable before it is used.